Makefile

```diff
@@ -41,8 +44,14 @@ binary-local:
 build-container:
 	docker build ${DOCKER_BUILD_ARGS} -t "$(DOCKER_IMAGE)" .
 
+docs/%.1: docs/%.1.md
+	$(GO_MD2MAN) -in $< -out $@.tmp && touch $@.tmp && mv $@.tmp $@
+
+.PHONY: docs
+docs: $(MANPAGES_MD:%.md=%)
+
 clean:
-	rm -f skopeo
+	rm -f skopeo docs/*.1
 
 install: install-binary
 	install -m 644 man1/skopeo.1 ${MANINSTALLDIR}/man1/
```

Comment:
@jwhonce sure, but we also want to install the manpage, correct? Or maybe I could do that in a followup commit. Just that this upstreams some lines from the rpm spec file.

fwiw, both the man and markdown files end up in the `docs/` dir after running `make docs` from your commit, so `install -m 644 man1/skopeo.1 ${MANINSTALLDIR}/man1/` won't work.

Let me know if I'm missing something.